.gitignore

```diff
@@ -80,6 +80,7 @@ TAGS
 *.mo
 
 .vscode
+.idea
```

Comment:
You can ignore the files that your editor environment creates _in all git repositories_ by creating a "global gitignore file", so that you don't have to add this to every project you collaborate on.  https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Thx!